src/main/java/com/github/bpiatek/bbghbackend/ninetyminutes/domain/PlayerCreator.java

```diff
@@ -0,0 +1,80 @@
+package com.github.bpiatek.bbghbackend.ninetyminutes.domain;
+
+import static java.util.regex.Pattern.DOTALL;
+import static java.util.regex.Pattern.quote;
+
+import com.github.bpiatek.bbghbackend.model.player.Player;
+import lombok.AllArgsConstructor;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by Bartosz Piatek on 12/10/2020
+ */
+@Service
+@AllArgsConstructor
+class PlayerCreator {
+
+  private static final Pattern PATTERN = Pattern.compile(quote("<td><b>") + "(?<skip>.*?)" + quote("</b></td>") +
+                                                         quote(" <td>") + "(?<value>.*?)" + quote("</td>"), DOTALL);
+
+  private final TextToLocalDateTimeParser toLocalDateTimeParser;
+   
+  Player createFromHtml(String html, Integer urlId) {
```

Comment:
z tej metody nie jestem dumny, ale juz trudno. Wyglada na to, ze dziala :D

Reply:
Testy przechodzi.